examples/Air-passengers-example.ipynb

```diff
@@ -6,26 +6,25 @@
    "metadata": {
```

Comment:
Will be easier to keep output here as this renders on the github page for people that browse the repo. Also much less diff to store for git

Reply:
Does this mean it's important to run the notebook before pushing it to GitHub (assuming it has changed)?

Reply:
@pennfranc I think it would be nice yes. However notebooks are kind of a pain to maintain in the repo overall... it'd be nice to find a better setup for them somehow.